websites/A/AnimeDigitalNetwork/presence.ts

```diff
@@ -11,16 +11,11 @@ presence.on("UpdateData", async () => {
 
   if (document.location.pathname.includes("video") && video) {
     if (video && !isNaN(video.duration)) {
-      const title = document.querySelector(".adn-player-header a").textContent,
-        subtitle = document.querySelector(".adn-player-header span")
-          .textContent,
-        timestamps = presence.getTimestamps(
-          Math.floor(video.currentTime),
-          Math.floor(video.duration)
-        ),
-        data: PresenceData = {
+      const t = document.querySelector("#root > div > div > div.sc-qXJnB.ijvXfJ > div > div:nth-child(1) > div.sc-jOdeeR.sc-kHIeKe.fmWHUu > div:nth-child(1) > div > div > h1 > a").textContent.toLowerCase(),
+      title = t2.charAt(0).toUpperCase() + t2.slice(1), 
```

Comment:
just wandering where you get t2 from, cause I can't find it...

Reply:
oh lmao, i forgot to change variable name when I suggested changes 😂

Reply:
![ADN10](https://user-images.githubusercontent.com/42322979/114664936-b3f80680-9cfc-11eb-8caf-8bcfcc3c3678.jpg)
![ADN9](https://user-images.githubusercontent.com/42322979/114664939-b5293380-9cfc-11eb-9ffd-7f3bb4455123.jpg)